### PR TITLE
Miletone 1: Model 1

### PR DESCRIPTION
**Pull request by henrycode460**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (keroloussamy) [inline .gitignore:48]: - [ ] Kindly this file its name should be `linters.yml`, and it has to be inside folder called `workflows`, and this folder inside another folder called `.github`. to be like `.github/workflows/linters.yml`.
- [ ] Kindly the `.gitignore` file should have only two things the `test.md` file and `node_modules` directory. Just type their names inside it.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @henrycode460, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯


- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._

- [x] Kindly add a descriptive pull request (description, title, branch-name) of what you have done in this pull request so that others can read what is being worked on. Check out this [link ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)to learn how to do this and then add a descriptive title and description to your pull request.
You can do this by clicking on the Edit button and adding the title and description.
Your PR
image

- [ ] Right now your branch called newBranch, Please rename it to be descriptive. To rename your branch follow this [article](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch)




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (henrycode460) [reply .gitignore:48]: can i do it over?

Comment (henrycode460) [reply .gitignore:48]: @keroloussamy Is there hint on how i can do the last part. The one that require me change the file name and placing file in another directory

**Pull request closed without merging**